src/lib/Preview.js

```diff
@@ -1245,6 +1267,9 @@ class Preview extends EventEmitter {
         this.viewer.addListener('error', this.triggerError);
         this.viewer.addListener(VIEWER_EVENT.default, this.handleViewerEvents);
         this.viewer.addListener(VIEWER_EVENT.metric, this.handleViewerMetrics);
+        if (this.options.enableAdvancedContentInsights) {
```

Comment:
There's already a handler for the various viewer events `handleViewerEvents`, is there a reason to separate it out here?